depthai_nodes/parsing_neural_network.py

```diff
@@ -127,6 +127,13 @@ def setBackend(self, setBackend: str) -> None:
         """Sets the backend of the NeuralNetwork node."""
         self._nn.setBackend(setBackend)
 
+    def getParser(self, parserID: int = 0) -> BaseParser:
```

Comment:
What are your thoughts on having default value here? The other option is that you have to explicitly speciffy the parserID but I agree that in 90% of the cases users are dealing with 1parser NNs so this would be then the desired behaviour. CC: @jkbmrz @kkeroo @aljazkonec1

Reply:
We could also set the default value to `None` and if no value is provided:
- return the parser at zero index if there is only one parser available,
- error out if there is multiple parsers available to avoid mistakes.

BTW, is `parserID` a good naming? I think `parser_index` would be more appropriate.

Reply:
Having an int value set to None would work but could cause issues down the line when porting to depthai. The function name `getParser` implies that only one is returned and I would just update the docstring such that it explicitly states which one is returned ( i.e. something like: The first parser is returned by default if multiple parsers available).

Regarding the naming, I would switch to snake case (to be consistent with other variables/attributes). `index` does make slightly more sense.

Reply:
I would leave the default option, so calling `getParser()` returns the first one. I'm fine with both ways of doing it (Petros' way and Jakob's way).